cli/admin/admin.go

```diff
@@ -0,0 +1,92 @@
+package admin
+
+import (
+	"fmt"
+
+	"github.com/spf13/pflag"
+
+	"github.com/apex/log"
+	"github.com/tarantool/cartridge-cli/cli/context"
+	"github.com/tarantool/cartridge-cli/cli/project"
+)
+
+const (
+	adminListFuncName = "__cartridge_admin_list"
+	adminHelpFuncName = "__cartridge_admin_help"
+	adminCallFuncName = "__cartridge_admin_call"
+)
+
+func List(ctx *context.Ctx) error {
+	if ctx.Project.Name == "" {
+		return fmt.Errorf("Please, specify application name using --name")
+	}
+
+	if err := project.SetSystemRunningPaths(ctx); err != nil {
+		return fmt.Errorf("Failed to get default paths: %s", err)
+	}
+
+	log.Debugf("Run directory is set to: %s", ctx.Running.RunDir)
+
+	// XXX: or use passed socket-path
+	conn, err := getAvaliableConn(ctx)
+	if err != nil {
+		return fmt.Errorf("Failed to connect to application instance socket: %s", err)
+	}
+	defer conn.Close()
+
+	if err := adminFuncList(conn); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func Help(funcName string, flagSet *pflag.FlagSet, ctx *context.Ctx) error {
+	if ctx.Project.Name == "" {
+		return fmt.Errorf("Please, specify application name using --name")
+	}
+
+	if err := project.SetSystemRunningPaths(ctx); err != nil {
+		return fmt.Errorf("Failed to get default paths: %s", err)
+	}
+
+	log.Debugf("Run directory is set to: %s", ctx.Running.RunDir)
+
+	// XXX: or use passed socket-path
+	conn, err := getAvaliableConn(ctx)
+	if err != nil {
+		return fmt.Errorf("Failed to connect to application instance socket: %s", err)
+	}
+	defer conn.Close()
```

Comment:
This code is the same as in function below. I suppose it could be placed in separate helper

Reply:
Restructured calling admin functions.